agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2367,6 +2368,33 @@ bool backhaul_manager::handle_1905_topology_query(ieee1905_1::CmduMessageRx &cmd
         }
     }
 
+    for (const auto &neighbor_device : m_1905_neighbor_devices) {
```

Comment:
I want a big comment here that explains that our 1905 neighbour device TLV is not entirely correct (but close enough).
```
        // The tlv1905NeighborDevice we construct here is not entirely correct, but close enough.
        // We should create a tlv1905NeighborDevice for every local interface on which a 1905 neighbor
        // is visible, and then add a list of neighbors visible on that interface. However, since we don't
        // keep track of the incoming interface of the received topology discovery messages, we can't do that.
        // Instead, we model it as a single interface, the bridge interface, behind which all neighbors are
        // visible, with a bridge between them. This is accurate enough to allow us to work in practice.
```

Also, I think we should create an issue (for M3 or M5) to correct this.

Reply:
added